### ISSUE TITLE
`yii\base\ErrorException` on PHP 7.4

### ISSUE THREAD

**Issue by cedx** [cedx/yii2-mustache#7]
The `yii\base\ErrorException` exception is raised when using the `ViewRenderer::render()` method on PHP 7.4rc5:

```
yii\mustache\ViewRenderer
 ✘ ->render()
   │
   │ yii\base\ErrorException: Trying to access array offset on value of type null
   │
   │ C:\Projets\php\yii2-mustache\vendor\mustache\mustache\src\Mustache\Parser.php:278
   │ C:\Projets\php\yii2-mustache\vendor\mustache\mustache\src\Mustache\Parser.php:103
   │ C:\Projets\php\yii2-mustache\vendor\mustache\mustache\src\Mustache\Parser.php:43
   │ C:\Projets\php\yii2-mustache\vendor\mustache\mustache\src\Mustache\Engine.php:783
   │ C:\Projets\php\yii2-mustache\vendor\mustache\mustache\src\Mustache\Engine.php:808
   │ C:\Projets\php\yii2-mustache\vendor\mustache\mustache\src\Mustache\Engine.php:738
   │ C:\Projets\php\yii2-mustache\vendor\mustache\mustache\src\Mustache\Engine.php:657
   │ C:\Projets\php\yii2-mustache\vendor\mustache\mustache\src\Mustache\Engine.php:236
   │ C:\Projets\php\yii2-mustache\lib\ViewRenderer.php:99
   │ C:\Projets\php\yii2-mustache\test\ViewRendererTest.php:28
   │ C:\Projets\php\yii2-mustache\vendor\cedx\phpunit-expect\lib\functions.php:30
   │ C:\Projets\php\yii2-mustache\test\ViewRendererTest.php:33
```

**Comment by cedx** [comment]
This issue can't be resolved by this project. The bug is in the `mustache` project:
https://github.com/bobthecow/mustache.php/issues/348

**Comment by cedx** [comment]
Fixed by version 2.13.0 of `mustache` package: just do a `composer update`.
